src/DotNetWorker.Core/Context/DefaultFunctionContext.cs

```diff
@@ -54,6 +57,8 @@ public override IServiceProvider InstanceServices
 
         public override TraceContext TraceContext => _invocation.TraceContext;
 
+        public override CancellationToken CancellationToken => _cancellationToken;
```

Comment:
Small nit: consider a descriptive property name around what this cancellation token is cancelling exactly. Is it `InvocationCancelled`? Or is the whole worker being cancelled?

Reply:
I believe the goal here is to have parity with the existing [cancellation parameter ](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library?tabs=v2%2Ccmd#cancellation-tokens) we have with in-proc. Since this is exposed to customers in the context, I think it makes more sense to keep the name they would expect. I think the documentation should be clear about it being a function invocation cancel

Reply:
I don't feel that strongly about this, so I am fine if you leave it as is. However, the concern I have is specifically about naming for property vs parameter cancellation token. Parameter (like in proc) the name of `cancellation` or `cancellationToken` is totally fine - it is obvious what the cancellation applies to: it applies to this call chain. A cancellation token as a property it is not as clear what it applies to. Which is were a more explicit name indicating what it applies to comes in handy.

Reply:
Ah gotcha, makes sense!